Xkcd-Comics/ContentViews/ExplanationSheet.swift

```diff
@@ -7,12 +7,12 @@
 
 import SwiftUI
 
-struct ExplanationSheet: View {    
-    var explainationSheetVM = ExplanationSheetVM()
-    var num: Int
+struct ExplanationSheet: View {
+    let viewModel: ExplanationSheetVM
+    let num: Int
```

Comment:
`num` should be injected into the view model instead of into the view. 

All business logic is typically the view model's domain. The view should know as little as possible about logic and data and only focus on how things look.